lumicks/pylake/force_calibration/tests/test_power_model.py

```diff
@@ -79,4 +79,4 @@ def test_fit_analytic_curve():
     ref = [0.0396382, 0.0389208, 0.03691641, 0.03399826, 0.03061068, 0.02713453]
     fit = fit_analytical_lorentzian(ps)
     np.testing.assert_allclose(fit.ps_fit.frequency, np.arange(0, 60, 10))
-    np.testing.assert_allclose(fit.ps_fit.power, ref)
+    np.testing.assert_allclose(fit.ps_fit.power, ref, 1e-5)
```

Comment:
maybe add the keyword to the final argument? seems like we usually do this and it would make it clear in the code, don't have to look up the docs if you can't remember if `atol` or `rtol` comes first

Reply:
Yeah, good point. Done! 👍